release notes/v0.46.0.md

```diff
@@ -8,51 +8,23 @@ k6 `v0.46` is here 🎉! This release includes:
-- [browser#945](https://github.com/grafana/xk6-browser/pull/945) Refactors `browser.contexts()` method to `browser.context()`. This is a required change after the modifications done to only allow a single `browserContext` in [browser#929](https://github.com/grafana/xk6-browser/pull/929).
 
-### (_optional h3_) `<big_breaking_change>` `#pr`
-
-## New features
-
-### Loki log output sending additional headers [#3227](https://github.com/grafana/k6/pull/3227)
-
-k6 has been able to send logs to loki for nearly 3 years since [v0.28.0](https://github.com/grafana/k6/releases/tag/v0.28.0) but didn't support any way to authenticate.
-
-Now it can be configured to send additional headers on every request.
-
-This is being configured with the new `header` config option similar to `label`:
-
-```
-k6 --log-output=loki=http://example.org,header.X-My-Header=123,header.Authorization=mytoken ...
-```
+### Browser
 
-Will now send the header `X-My-Header` with value `123` and `Authorization` with `mytoken`.
-
-Thanks to @federicotdn for doign the changes
+In this release, xk6-browser extension version is bumped up to `v1.0.2`, as it includes multiple breaking changes in relation with options, browser lifecycle and metrics.
 
-### Cloud Traces [#3100](https://github.com/grafana/k6/pull/3100), [#3202](https://github.com/grafana/k6/pull/3202)
+**Options** `devtools`, `env` and `proxy` are deprecated ([browser#868](https://github.com/grafana/xk6-browser/pull/868), [browser#870](https://github.com/grafana/xk6-browser/pull/870), [browser#872](https://github.com/grafana/xk6-browser/pull/872)). Additionally, browser `launch`/`connect` options are no longer defined in the corresponding JS API methods, instead the test execution related options are now defined inside the browser scenario options (see [#3036](https://github.com/grafana/k6/pull/3036)), and the other more "environmental options", such as `headless`, `debug`, `executablePath`, are set as ENV vars. Also `slowMo` option is no longer supported, although it might be supported again in the future through a different API ([browser#876](https://github.com/grafana/xk6-browser/pull/876)).
 
-This release supports the new integration between k6 and Tempo in the cloud. Grafana Cloud k6 and Grafana Cloud Traces customers will be able to start their traces in k6 using the existing `k6/experimental/tracing` module to enrich their test run analysis page with metrics and aggregations from tracing data.
+**Metrics** also suffer a few modifications. The Web Vitals metrics are renamed to use `browser_` prefix and short namings (e.g.: `webvital_first_input_delay` -> `browser_web_vital_fid`) ([browser#885](https://github.com/grafana/xk6-browser/pull/885), [browser#903](https://github.com/grafana/xk6-browser/pull/903)), and the rating metric is removed, as it is now set as a label in the corresponding Web Vitals metrics ([browser#915](https://github.com/grafana/xk6-browser/pull/915))
```

Comment:
I think the tone needs to soften here a bit, how about something like: `**Metrics** also went through a few changes.`?

Reply:
Right, I was not linking the word `suffer` but could not come up with a different word/expression at the moment :+1: